pkg/source/archive_test.go

```diff
@@ -162,3 +163,13 @@ func TestHandleZip(t *testing.T) {
 		})
 	}
 }
+
+func normalizeNewlines(d []byte) []byte {
+	CR := []byte{13}
```

Comment:
declare linebreak as a constant and also as `/n` instead of byte array. It makes the code more readable